core/sql/regress/tools/runregr_hive.ksh

```diff
@@ -225,43 +225,8 @@ skippedfiles=
 testfiles="$prettyfiles"
 prettyfiles=
 
-# skip these tests on all platforms.
-skipTheseTests=""
-
-# skip these tests for SQLMX and SQLMP tables on NSK platform
-if [ $nsk -eq 1 ]; then
-  skipTheseTests="TEST*"
-fi
-
-# skip these tests for RELEASE ONLY tests on NSK
-#if [ "$BUILD_FLAVOR" = "RELEASE" ]; then
-#  skipTheseTests="$skipTheseTests TESTxxx"
-#fi
-
-# skip these tests for DEBUG ONLY tests on NSK
-#if [ "$BUILD_FLAVOR" = "DEBUG" ]; then
-#  skipTheseTests="$skipTheseTests TESTxxx"
-#fi
-
-# skip these tests for NT
-if [ `uname` = "Windows_NT" ]; then
-  skipTheseTests="TEST*"
-fi
-
-
-#skip these tests for Linux
-#if [ `uname` = "Linux" ]; then
-#  skipTheseTests="$skipTheseTests TESTxxx"
-#fi
-
-#skip these tests when seabase eq 1.
-if [ "$seabase" -ne 0 ]; then
-  if [ "$seabase" -eq 1 ]; then
-    skipTheseTests="TEST004"
-  fi
-fi
-#skipTheseTests="TEST020"
-skipTheseTests=
+# skip these tests
+skipTheseTests="TEST020"
```

Comment:
changes that test020 is testing are still being worked on.
once they are complete, will enable this test.